samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -233,7 +233,12 @@ static const char link_ncellmeas_usage_str[] =
 	"                      New cell measurement is done everytime when current cell changes.\n"
 	"   --cancel,          Cancel/Stop neighbor cell measurement if still ongoing\n"
 	"   --search_type,     Used search type:\n"
-	"                      'default', 'ext_light' or 'ext_comp.'\n"
+	"                      'default', 'ext_light', 'ext_comp', 'gci_1', 'gci_2' and 'gci_3'.\n"
```

Comment:
If the GCI search types in LTE LC are renamed as I suggested, these parameters should also be changed `gci_default`, `gci_ext_light` and `gci_ext_comp` (or something similar).

Reply:
Changed